src/node.h

```diff
@@ -463,14 +465,16 @@ using StartExecutionCallback =
 
 // TODO(addaleax): Deprecate this in favour of the MaybeLocal<> overload.
 NODE_EXTERN void LoadEnvironment(Environment* env);
+// The `InspectorParentHandle` arguments here are ignored and not used.
+// For passing `InspectorParentHandle`, use `CreateEnvironment()`.
```

Comment:
i assume these are left here for abi compat? should we issue a deprecation?

Reply:
Yeah – we don’t really have to be concerned about this, this hasn’t gone out in a release yet, but at the same time leaving them here and removing them in a semver-major PR is also very cheap and doesn’t hurt anybody.

(And embedders might be the main Node.js consumer group that doesn’t wait for releases.)